cluster/saltbase/salt/kube-addons/init.sls

```diff
@@ -87,6 +87,17 @@ addon-dir-create:
     - file_mode: 644
 {% endif %}
 
+{% if pillar.get('enable_cluster_ui', 'true').lower() == 'true' %}
```

Comment:
No, does it need to be? I thought that since the default was true, we didn't need to override it in files like https://github.com/GoogleCloudPlatform/kubernetes/blob/ffb846a284fd0fd125e331fb01b26a8d4650ea9c/cluster/gce/configure-vm.sh#L258

Does the variable also need to be declared somewhere?